src/Controls/tests/DeviceTests/Elements/SwipeView/SwipeViewTests.Android.cs

```diff
@@ -68,5 +67,48 @@ public async Task SwipeItemSizeInitializesCorrectly()
 				Assert.NotEqual(0, swipeItem.Width);
 			});
 		}
+		
+		[Fact(DisplayName = "SwipeView LogicalChildren Works Correctly")]
+		public async Task SwipeViewLogicalChildren()
```

Comment:
Can we run these tests on all three platforms?

Reply:
Sure thing, updated.